slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -121,12 +125,14 @@ class SLAMServiceImpl final : public SLAMService::Service {
     double rotation_weight = 1.0;
 
    private:
-    int starting_scan_number = 0;
     int picture_print_interval = 50;
```

Comment:
If you want to remove anything related to what will be implemented with `input_file_pattern`, then `picture_print_interval` should probably be deleted, too.